Mathlib/Algebra/Module/LocalizedModule.lean

```diff
@@ -1244,3 +1244,33 @@ end Algebra
 end IsLocalizedModule
 
 end IsLocalizedModule
+
+section Subsingleton
+
+variable {R M} [CommRing R] [AddCommGroup M] [Module R M]
```

Comment:
```suggestion
variable {R M : Type*} [CommRing R] [AddCommGroup M] [Module R M] {p : PrimeSpectrum R}
```

If it is easy, it is better to avoid level metavariables in `R` and `M`. 

Is there a reasonable to repeat `{p}` so much? Do you prefer the argument order?